src/main/resources/templates/appointments/add-trainer.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="https://www.thymeleaf.org/">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+<form method="post" action ="/trainers/add-trainer">
+    <label for="firstName">First Name</label>
+    <input id = "firstName" type="text" name="firstName"  >
+    <label for="lastName">Last Name</label>
+    <input id = "lastName" type="text" name="lastName" >
+    <label for="phone">Phone Number(xxx-xxx-xxxx)</label>
+    <input type="tel" id="phone" name="phone" pattern="[0-9]{3}-[0-9]{3}-[0-9]{4}">
```

Comment:
I'm not familar with this pattern. Could you explain this to me please?

Reply:
I think you wrote this as an example during studio, but since this appears to come direct from [w3schools](https://www.w3schools.com/tags/att_input_type_tel.asp), i'll answer.
This is standard regex syntax
`[0-9]` is the set digits from 0-9 followed by '{x}' where 'x' is the number of times a member of the set must appear in order to match. then '-' which is just itself.
https://www.oreilly.com/library/view/oracle-regular-expressions/0596006012/re13.html